ClosedXML/Excel/ConditionalFormats/Save/XLCFEndsWithConverter.cs

```diff
@@ -9,9 +9,9 @@ public ConditionalFormattingRule Convert(IXLConditionalFormat cf, int priority,
         {
             String val = cf.Values[1].Value;
             var conditionalFormattingRule = XLCFBaseConverter.Convert(cf, priority);
-
-            if (!cf.Style.Equals(XLWorkbook.DefaultStyle))
-                conditionalFormattingRule.FormatId = (UInt32)context.DifferentialFormats[cf.Style];
+            var cfStyle = cf.Style.Value;
+            if (!cfStyle.Equals(XLWorkbook.DefaultStyle))
```

Comment:
Should be ` if (!cfStyle.Equals(XLWorkbook.DefaultStyle.Value))` ?

Reply:
Good catch, thank you!